app/models/comments.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_create :update_comments_counter
+
```

Comment:
Hey there! Just a friendly suggestion for your code. If you're using the after_create option, it might be helpful to place the method you're calling after the private method. This way, it'll be easier to understand and follow the flow of your code. 🤓💻

Reply:
This is very helpful to know. I will make the change and be sure to keep this in mind in my next projects. Thank you very much.